src/Build/Graph/ProjectGraphNode.cs

```diff
@@ -28,7 +27,7 @@ internal ProjectGraphNode(Project project)
         /// <summary>
         /// Gets the evaluated project represented by this node in the graph.
         /// </summary>
-        public Project Project { get; }
+        public ProjectInstance Project { get; }
```

Comment:
After speaking offline, we decided ProjectInstance would be better, at least for now since DefaultTargets isn't exposed on Project.